Makefile

```diff
@@ -79,6 +79,9 @@ cross: build ## cross build the binaries for darwin, freebsd and\nwindows
 win: build ## cross build the binary for windows
 	$(DOCKER_RUN_DOCKER) hack/make.sh win
 
+darwin: build ## cross build the binary for OS X
+	$(DOCKER_RUN_DOCKER) hack/make.sh darwin
```

Comment:
Apparently it was added as part of https://github.com/docker/docker/pull/20478, but I'm not sure *why*, and if its really needed